src/Builder/RouteInfo.php

```diff
@@ -45,15 +46,17 @@ public function getName(): string
     }
 
     /**
-     * Is visible?
+     * Whether it should be exposed to the static site builder or not.
+     * If not visible, the route will be ignored by the {@link Builder},
+     * which won't generate a static file for matching urls.
      */
     public function isVisible(): bool
     {
-        return $this->route->getOption('visible') ?? $this->name[0] !== '_';
+        return $this->route->getOption('content')['visible'] ?? $this->name[0] !== '.';
```

Comment:
J'ai gardé `visible`, mais il y aurait peut-être mieux comme terme?

A côté de ça, je propose de plutôt ignorer les routes commençant par un `.`, répliquant la convention utilisée pour les [services cachés](https://symfony.com/blog/new-in-symfony-4-1-hidden-services) dans Symfony.

Reply:
`ignore: true` ?

Reply:
Done !